tslib.js

```diff
@@ -57,6 +58,13 @@ var __generator;
         return t;
     };
 
+    __rest = (this && this.__rest) || function (s, e) {
+        var t = {};
+        for (var p in s) if (Object.prototype.hasOwnProperty.call(s, p) && !e.indexOf(p))

```

Comment:
Would the fix be `getOwnPropertyDescriptors`? I don't know that much about `Symbol` keys.

I don't think this PR is the place to fix it regardless, because `__extends` and `__assign` already use `for .. in`.